SwrveSDK/src/google/java/com/swrve/sdk/gcm/SwrveGcmHandler.java

```diff
@@ -33,39 +35,26 @@ protected SwrveGcmHandler (Context context, SwrveGcmIntentService swrveGcmServic
         this.swrveGcmService = swrveGcmService;
     }
 
+    @Deprecated
     @Override
     public boolean onHandleIntent(Intent intent, GoogleCloudMessaging gcm) {
+        return false;
```

Comment:
Would removing it from the interface and class be a better option? If we only deprecate it customers might update and not notice the deprecated warning, then push stops working for no apparent reason.